promoted-ruby-client.gemspec

```diff
@@ -0,0 +1,43 @@
+
+lib = File.expand_path("../lib", __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "promoted/ruby/client/version"
+
+Gem::Specification.new do |spec|
+  spec.name          = "promoted-ruby-client"
+  spec.version       = Promoted::Ruby::Client::VERSION
+  spec.authors       = ["dev-ham"]
+  spec.email         = ["devhammad.masood@gmail.com"]
+
+  spec.summary       = 'A Ruby Client to contact Promoted APIs.'
+  spec.description   = 'This is primarily intended to be used when logging Requests and Insertions on a backend server.'
+  spec.homepage      = 'https://github.com/promotedai/promoted-ruby-client'
+  spec.license       = 'MIT'
+
+  # Prevent pushing this gem to RubyGems.org. To allow pushes either set the 'allowed_push_host'
```

Comment:
Is it because we don't want to push yet?

Reply:
yep